tests/AsyncTest.php

```diff
@@ -44,4 +83,44 @@ public function testInstanceOfInterface() {
     $xhp = <async:test><b>BE BOLD</b></async:test>;
     $this->assertInstanceOf(XHPAwaitable::class, $xhp);
   }
+
+  public function parallelizationContainersProvider() {
+    return [
+      [<test:xfrag-wrap />],
+      [<test:async-xfrag-wrap />],
+    ];
+  }
+
+  /**
+   * @dataProvider parallelizationContainersProvider
+   */
+  public function testParallelization(:x:element $container) {
+    :async:par-test::$log = Vector { };
+
+    $a = <async:par-test label="a" />;
+    $b = <async:par-test label="b" />;
+    $c = <async:par-test label="c" />;
+
+    $container->replaceChildren([$b, $c]);
```

Comment:
What happens if you replace this with `$container->replaceChildren(<div>{$b}{$c}</div>);`? Does it still pass?